medical_prescription_state_verify/models/medical_prescription_order_line.py

```diff
@@ -29,13 +26,19 @@ def write(self, vals, ):
             ValidationError: When a write is not allowed due to being in a
                 protected state
         """
-        for rec_id in self:
-            if rec_id.state_type == 'verified':
-                raise exceptions.ValidationError(_(
-                    'You cannot edit this value after its parent Rx has'
-                    ' been verified. Please either cancel it, or mark it as'
-                    ' an exception if manual reversals are required. [%s]' %
-                    rec_id.prescription_order_id.name
-                ))
-
-            return super(MedicalPrescriptionOrderLine, self).write(vals)
+        if self.prescription_order_id.stage_id.is_verified:
+            if vals.get('stage_id'):
+                stage_id = self.env['base.kanban.stage'].browse(
+                    vals['stage_id']
+                )
+                if stage_id.name == "Hold":
+                    return super(
+                        MedicalPrescriptionOrderLine, self
+                    ).write(vals)
+            raise ValidationError(_(
+                'You cannot edit this value after its parent Rx has'
+                ' been verified. Please either cancel it, or mark it as'
+                ' an exception if manual reversals are required. [%s]' %
+                self.prescription_order_id.name
```

Comment:
The string interpolation should be outside of the translation - you don't want to translate the interpolation. `_('text %s') % var`